include/libopencmsis/core_cm3.h

```diff
@@ -26,7 +26,7 @@
  * libopencm3 ones, "volatile" is all that's missing. */
 #define __IO volatile
 #define __O volatile
-#define __I volatile
+#define __I volatile const
```

Comment:
this seems very out of scope for this commit! (and needs further explanation)

Reply:
this seems to be an oversight..